lambdas/service/app.py

```diff
@@ -1384,7 +1384,12 @@ def generate_manifest(event, context):
                 more information refer to https://dss.data.humancellatlas.org
                 under `GET /files/{uuid}`. The only mandatory forwarded
                 parameter is `replica`, for which Azul only supports AWS.
-            ''')
+            '''),
+        params.query(
+            'token',
+            schema.optional({'type': 'string'}),
```

Comment:
There is a shortcut for strings. If you borrowed this from somewhere else, that place should also use the shortcut.